packages/commonwealth/server/routes/status.ts

```diff
@@ -35,10 +34,13 @@ const status = async (
           {
             model: models.ChainNode,
           },
+          {
+            model: models.Contract,
+            required: false,
```

Comment:
Should we require ChainNode here as well?

Reply:
we could!